Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -138,7 +138,7 @@ def test_select_font_key(self):
         self.assertEqual(dialog.set_font_sample.called, 2)
 
     def test_select_font_mouse(self):
-        # Click on item should select that item.
+        "Click on item should select that item."
```

Comment:
I tried this once and stopped when I discovered that running a file with unittest.main(verbosity=2) prints the docstrings, as in this example where I added one.  Ditto for python -m test.test_idle -v.
`test_font (__main__.FontTabTest) ... ok`
`test_set_sample (__main__.FontTabTest)`
`Set_font_sample also sets highlight_sample. ... ok`
`test_tabspace (__main__.FontTabTest) ... ok`
I decided I did not like it and reverted the couple of examples.  

Verbosity 1 prints a '.' for each successful test.  0 prints nothing until the end.

With the effect understood, I am willing to discuss a change, with at least Cheryl included.
In the meanwhile, please revert (you can use pencil edit upper right).